recipes/llvm-core/all/conandata.yml

```diff
@@ -0,0 +1,41 @@
+sources:
+    "11.0.0":
+        url: https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/llvm-11.0.0.src.tar.xz
+        sha256: 913f68c898dfb4a03b397c5e11c6a2f39d0f22ed7665c9cefa87a34423a72469
+
+    "10.0.1":
+        url: https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.1/llvm-10.0.1.src.tar.xz
+        sha256: c5d8e30b57cbded7128d78e5e8dad811bff97a8d471896812f57fa99ee82cdf3
+
+    "10.0.0":
+        url: https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/llvm-10.0.0.src.tar.xz
+        sha256: df83a44b3a9a71029049ec101fb0077ecbbdf5fe41e395215025779099a98fdf
+
+    "9.0.1":
+        url: https://github.com/llvm/llvm-project/releases/download/llvmorg-9.0.1/llvm-9.0.1.src.tar.xz
+        sha256: 00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a
```

Comment:
Can you keep only the latest version ? We will add other versions when the recipe is stabilized and merged.

Reply:
Sure!

Reply:
I think I'm resolving this conversation as the suggested changes were made.